apps/docs/src/app/platform/component-docs/platform-forms/platform-textarea/platform-textarea-examples/platform-textarea-basic-example.component.html

```diff
@@ -56,7 +56,7 @@
         <div fd-form-item>
             <label fd-form-label for="textarea-1">Textarea without Platform Forms</label>
             <fdp-textarea
-                id="textarea-1"
+                [id]="'textarea-1'"
```

Comment:
what this change is for?

Reply:
Label is not associated with native textarea, because there're 2 elements with the same `id`: `fdp-textarea` and underlying native control. Basically it's the same issue that we have with `title` binding all over the place. Discussed this with @Betrozov and we decided to create a separate issue to handle all similar cases for all components https://github.com/SAP/fundamental-ngx/issues/7297

Reply:
I see, it starts working with this binding but this is so not obvious )

Reply:
Yeah, totally agree, we need to rename such inputs everywhere